config/locales/en.yml

```diff
@@ -788,64 +806,50 @@ en:
     website: 'Trust website'
     school_visits_heading: 'Arranging a visit to %{school}'
   subscriptions:
-    back_to_search_results: 'Return to your search results'
     benefits:
       title: Subscribers
       list:
         - get an email when new jobs match their search
         - can unsubscribe at any time
         - receive no more than 1 email a day
-    footer: Teaching Vacancies is a free job-listing service from the Department for Education. It helps schools to save money on recruitment and you to take the next step in your career.
+    confirm:
+      back_to_search_results: 'Return to your search results'
+      body: We sent an email confirmation to %{email}.
+      frequency_label: 'You will receive alerts '
+      header: Your email subscription has started
+      next_step_details: 'We’ll email you details of any new teaching jobs that match the following criteria:'
+      next_step: What happens next
+      title: Subscription confirmation
+      unsubscribe: You can unsubscribe by following the link at the bottom of these emails.
+    confirm_update:
+      back_to_search_results: 'Return to your search results'
+      body: We sent an email confirmation to %{email}.
+      frequency_label: 'You will receive alerts '
+      header: Your job alert has been updated
+      next_step_details: 'We’ll email you details of any new teaching jobs that match the following criteria:'
+      next_step: What happens next
+      title: Subscription update
+      unsubscribe: You can unsubscribe by following the link at the bottom of these emails.
```

Comment:
frequency_label, next_step, and unsubscribe are duplicated - is there a reason?

Reply:
I can make the extra effort of removing the duplication, but I prefer to have all the keys in a view local to that view, so when you change something you know what you are changing on that specific page